appeals/web/src/server/appeals/appeal-details/representations/common/render-select-rejection-reasons.js

```diff
@@ -22,6 +22,7 @@ export const renderSelectRejectionReasons = (contentMapper) => async (request, r
 			currentRepresentation,
 			rejectionReasons,
 			session,
+			'rejectIpComment',
```

Comment:
You'll need `renderSelectRejectionReasons` to also take a sessionKey arg so it passes this down. There shouldn't be reference to IP comments in the common folder

Reply:
We can go through this together if you want, I've got it in my PR